### PR TITLE
added the use for scikit-learn

### PR DESCRIPTION
**Pull request by AV25242**

### CODE REVIEW

File: docs/machine-learning/tutorials/python-ski-rental-linear-regression-train-model.md [hunk 1]
```diff
@@ -39,13 +39,20 @@ In [part four](python-ski-rental-linear-regression-deploy-model.md), you'll lear
 
 * Part three of this tutorial assumes you have completed [part one](python-ski-rental-linear-regression.md) and its prerequisites.
 
+! pip install scikit-learn
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/tutorials/python-ski-rental-linear-regression-train-model.md]:
```suggestion
* Part three of this tutorial assumes you have completed [part one](python-ski-rental-linear-regression.md) and its prerequisites, including [installing the necessary Python packages](python-ski-rental-linear-regression.md#prerequisites).
```

File: docs/machine-learning/tutorials/python-ski-rental-linear-regression-train-model.md [hunk 2]
```diff
@@ -39,13 +39,20 @@ In [part four](python-ski-rental-linear-regression-deploy-model.md), you'll lear
...
 ## Train the model
 
 In order to predict, you have to find a function (model) that best describes the dependency between the variables in our dataset. This called training the model. The training dataset will be a subset of the entire dataset from the pandas data frame `df` that you created in part two of this series.
 
 You will train model `lin_model` using a linear regression algorithm.
 
 ```python
+
+from sklearn.linear_model import LinearRegression
+from sklearn.metrics import mean_squared_error
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/tutorials/python-ski-rental-linear-regression-train-model.md:52]: should these be `sklearn` or `scikit-learn`, @AV25242?

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]
Please review two comments, @AV25242

**Comment by Court72** [comment]
@AV25242 - Can you respond to the requested changes?
 
@WilliamDAssafMSFT - After the changes are made, please add a `#sign-off` comment to signal that the PR is ready to merge.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/machine-learning/tutorials/python-ski-rental-linear-regression-train-model.md (hunk 2 above)
Comment (AV25242) [reply docs/machine-learning/tutorials/python-ski-rental-linear-regression-train-model.md:52]: The class name has not changed, only the package name has changed. So the code would remain the same.

**Review by WilliamDAssafMSFT — APPROVED** [review]

**Comment by WilliamDAssafMSFT** [comment]
#sign-off
